datumaro/plugins/data_formats/datumaro_binary/format.py

```diff
@@ -13,4 +17,13 @@ class DatumaroBinaryPath:
     ANNOTATION_EXT = ".datumaro"
     IMAGE_EXT = ".jpg"
     MASK_EXT = ".png"
-    SIGNATURE = "signature:datumaro_binary"
+    SIGNATURE = _SIGNATURE
+    SIGNATURE_LEN = len(_SIGNATURE)
+    ENCRYPTION_FIELD_LENGTH = 512
```

Comment:
why this should be fixed?

Reply:
I removed ENCRYPTION_FIELD_LENGTH which will not be used. And, the change about signature is because I want to obtain the constant `SIGNATURE_LEN` from `SIGNATURE` at class variable initialization.